typescript/infra/config/environments/mainnet3/warp/configGetters/formConfigGetters.ts

```diff
@@ -0,0 +1,65 @@
+import { ethers } from 'ethers';
+
+import { ChainMap, HypTokenRouterConfig, TokenType } from '@hyperlane-xyz/sdk';
+import { Address } from '@hyperlane-xyz/utils';
+
+import {
+  RouterConfigWithoutOwner,
+  tokens,
+} from '../../../../../src/config/warp.js';
+
+const safeOwners: ChainMap<Address> = {
+  ethereum: '0xec5ad23e29203301B2C1a765718Cc1de7A8d3FbF',
+  form: '0x41B624412B529409A437f08Ef80bCabE81053650',
+};
+
+export const getEthereumFormUSDCWarpConfig = async (
+  routerConfig: ChainMap<RouterConfigWithoutOwner>,
+): Promise<ChainMap<HypTokenRouterConfig>> => {
+  const ethereum: HypTokenRouterConfig = {
+    mailbox: routerConfig.ethereum.mailbox,
```

Comment:
can we do the
```
...routerConfig.ethereum,
...getOwnerConfigForAddress(safeOwners.ethereum),
```
pattern that we've been generally using instead of specifying `mailbox` and `owner`